spec/Appendix B -- Grammar Summary.md

```diff
@@ -1,6 +1,6 @@
 # B. Appendix: Grammar Summary
 
-SourceCharacter :: /[\u0009\u000A\u000D\u0020-\uFFFF]/
+SourceCharacter :: "Any Unicode code point"
```

Comment:
Also note that allowing `U+0000` (NUL) will be interesting for a number of implementations as well as security checks, since C string APIs will assume that NUL is the end of a string.